### ISSUE TITLE
Visual Studio 2015 Build Tools

### ISSUE THREAD

**Issue by manu-st** [playscale/stashed.io#11]
I saw the report for #4 but I think it refers to the 2017 tools as after installing stashed.io on a machine with just the 2015 ones installed the front-end entries of the dashboard does not report anything.

**Comment by jsmouret** [comment]
Yes indeed, the previous fix was for 2017 build tools.

Would it be possible to check what is in that registry folder on the machine?

`Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\VisualStudio\SxS\VS7`

I just uninstalled the regular VS2015 to install the build tools and it seems to populate the same registry keys, so I'm not quite sure.

Thanks for your help

**Comment by manu-st** [comment]
Here is what I have. Note that I do not have a VS7 entry, just a VC7 one:
```
Windows Registry Editor Version 5.00

[HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\VisualStudio\SxS\VC7]
"FrameworkDir32"="C:\\Windows\\Microsoft.NET\\Framework\\"
"FrameworkDir64"="C:\\Windows\\Microsoft.NET\\Framework64"
"14.0"="C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\"
"FrameworkVer32"="v4.0.30319"
"FrameworkVer64"="v4.0.30319"
```

**Comment by jsmouret** [comment]
Thanks! I'll get back to you soon

**Comment by jsmouret** [comment]
Fixed in 1.4.2
Enjoy!
